service/controller/resource/tccpn/testdata/case-0-basic-test-with-encrypter-backend-KMS.golden

```diff
@@ -10,13 +10,17 @@ Resources:
     Type: AWS::AutoScaling::AutoScalingGroup
     Properties:
       VPCZoneIdentifier:
-        - !Ref PrivateSubnetEuCentral1b
+        - 
```

Comment:
That looks broken.

Reply:
fixed, default controller context was missing some value